docs/developer-docs/web-apps/verifiable-credentials/relying-party.mdx

```diff
@@ -0,0 +1,213 @@
+
+### Step 2: Request a verifiable credential
+
+Once the user has logged into Internet Identity and the service is ready, the relying party service can request a verifiable credential for the user by sending a JSON RPC request to the issuer API. The user must be logged into Internet Identity for the request to be successful. Below is an example of a JSON RPC request to request a credential:
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "request_credential",
+  "params": {
+    "issuer": {
+        "origin": "https://employment-info.com",
+        "canisterId": "rwlgt-iiaaa-aaaaa-aaaaa-cai"
+    },
+    "credentialSpec": {
+        "credentialType": "VerifiedEmployee",
+        "arguments": {
+          "employerName": "XYZ Ltd."
+        }
+    },
+    "credentialSubject": "2mdal-aedsb-hlpnv-qu3zl-ae6on-72bt5-fwha5-xzs74-5dkaz-dfywi-aqe"
+  }
+}
+```
+
+In this request, the following parameters must be set:
+
+- `method`: `request_credential`: Uses the method for requesting a verifiable credential.
+
+- `params`: Configuration parameters to identify which credential is being requested.
+
+    - `issuer`: The information to identify which issuer API instance the request is sent to.
+
+        - `origin`: The origin URL of the issuer.
+
+        - `canisterId`: The canister ID of the issuer API. This field is optional.
+
+  - `credentialSpec`: The specification of the credential being requested.
+
+    - `credentialType`: The type of the requested credential.
+
+    - `arguments`: Key value pairs that are specific to the credential. This field is optional.
+
+  - `credentialSubject`: The principal of the user requesting the credential.
+
+  - `derivationOrigin`: An origin that should be used for principal derivation as an alternative to the client origin. This field is optional.
+
+
+Additional examples can be found in the [verifiable credentials specification](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md#examples).
+
+### Step 3: Receive a response
+
+If the JSON RPC request is successfully sent and received by the issuer API, Internet Identity will return the following JSON RPC response to the relying party:
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": {
+    "verifiablePresentation": "eyJQ..."
+  }
+}
+```
+
+In this response, `verifiablePresentation`is a JWT based verifiable presentation (VP) containing two credentials:
+
+1. A verifiable credential issued by Internet Identity that relates the `credentialSubject` to the user principal.
+
+2. A verifiable credential issued by the issuer to the user principal.
+
+The order of the credentials in the VP should match the order above, such that the credential relating to the `credentialSubject` to the user principal should come first.
```

Comment:
It would be great with an explanation of **why** things happen this why, not only **what** happens. 
- Why are two credentials returned and not only the one I requested?
- What is the purpose of the id alias connecting the two credentials?

Reply:
Good point. I thought this was explained in the specification, but it's not.

How about adding the following paragraph:
> The specification ensures unlinkability between the relying party and the issuer. That's why we need two credentials. The first credential proves the link between the relying party's principal and the issuer's. The second credential is the actual credential to the user's principal from the issuer's side.

@jessiemongeon1 @kristoferlund what do you think?

Could you add a placeholder for a diagram. I will create a new diagram which will help with this topic.

Reply:
Added paragraph and placeholder for diagram